axon-server-connector/src/test/java/org/axonframework/axonserver/connector/event/LegacyAxonServerEventStorageEngineTest.java

```diff
@@ -53,6 +62,30 @@ static void afterAll() {
         axonServerContainer.stop();
     }
 
+    @Test
+    void eventWithMultipleTagsIsReportedAsPartOfException() {
```

Comment:
Isn't this a test for the `AggregateBasedStorageEngineTestSuite` in general?

Reply:
I decided not to force all implementations to use specific wording in their exceptions. The TooManyTagsOnEventMessageException is specific to this implementation. The generic test just expects that the attempt is rejected with an IllegalArgumentException.